tasks/sanity-optional-label-check.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-optional-label-check
+spec:
+  params:
+    - name: sourceImage
+      description: the fully qualified image name
+      default: quay.io/cvpops/test-index:v4.10
+
+  steps:
+    - name: optional-label-check
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      script: |
+        skopeo inspect docker://$(inputs.params.sourceImage) > /tekton/home/optional_label_image_inspect.json
```

Comment:
nit: you can add to speed up and improve stability `--no-tags`

Reply:
done